hack/arktos-cni.rc

```diff
@@ -66,6 +66,12 @@ install_cni_bridge() {
     echo "done with bridge cni plugin installation"
 }
 
+cleanup_cni_mizar() {
+    sudo rm -rf /opt/cni/bin/
+    [ ! -d "${cni_conf_dir}" ] && sudo mkdir -p "${cni_conf_dir}"
+    [ ! -d "${cni_bin_dir}" ] && sudo mkdir -p "${cni_bin_dir}"
+    sudo find /etc/cni/net.d/ -type f -not -name '10-mizarcni.conf' -delete
```

Comment:
Does /opt/cni/bin/ = ${cni_bin_dir}? can we use var to replace hardcode here?

Reply:
Yes, use var to replace hardcode.